indigo_api/templates/indigo_api/akn/coverpage_act.html

```diff
@@ -44,7 +44,15 @@ <h2>{{ subt }}</h2>
         </li>
       {% endif %}
 
-      {% if document.commencement_date %}
+      {% if not document.work.commenced %}
+        <li class="commencement-date">
+          {% blocktrans %}Not yet commenced: This work has not come into force yet.{% endblocktrans %}
+        </li>
+      {% elif not document.commencement_date %}
+        <li class="commencement-date">
+          {% blocktrans %}Commencement uncertain: This work has likely commenced but a commencement date hasn't been captured.{% endblocktrans %}
```

Comment:
I'd like to reword this, I'm not sure how yet. I think we should avoid using "work" for general users because it can be a confusing term. But I'm not sure what other word to use.